src/Pages/BeerGoggles/BeerGoggles.js

```diff
@@ -1,9 +1,203 @@
 import React from "react";
+import "./BeerGoggles.css";
+import podIcon from "../../images/podIcon.webp";
+import discordIcon from "../../images/discordIcon.png";
+import youtubeIcon from "../../images/youtubeIcon.png";
+import XcommitteePic from "../../images/xcommittee.jpg";
 
 export default function BeerGoggles() {
+    //TODO: swap BeerGoggles image in for Xcom Image
     return (
-        <div>
-            <p>Beer Goggles</p>
+        <div className="beerGooglesContainer">
+            <img className="xcomHeader" src={XcommitteePic} alt="xcombanner" />
+            <div className="beerGooglesIntroContainer">
+                <div className="beerGooglesDescription">
+                    <p>
+                        A fun little experiment where one of the HUGE members
+                        picks a game they remember loving from their childhood
+                        and plays through the game in its entirety with another
+                        HUGE member as a co-pilot. They see if they still love
+                        the game after replaying it or if it was just nostalgia;
+                        hence the name Beer Googles.
+                    </p>
+                    <p>
+                        In this premiere episode, Ben and Tim set out on an epic
```

Comment:
you may want to give this paragraph some kind of header since it is specifically about the God of War series and not about Beer Goggles in general. So Maybe say something like
**Featured Episode: God of War**
then the paragraph about the episode